perf-tests/src/main/scala/AkkaHttpTapirMultiServer.scala

```diff
@@ -0,0 +1,65 @@
+package perfTests
+
+import akka.actor.typed.ActorSystem
+import akka.actor.typed.scaladsl.Behaviors
+import akka.http.scaladsl.Http
+import akka.http.scaladsl.server.Route
+
+import sttp.tapir._
+import sttp.tapir.server.akkahttp.AkkaHttpServerInterpreter
+import akka.http.scaladsl.server.Directives.{concat}
+
+import scala.io.StdIn
+import scala.concurrent.Future
+
+class AkkaHttpTapirMultiServer {
+  implicit val actorSystem = ActorSystem(Behaviors.empty, "akka-http")
+  implicit val executionContext = actorSystem.executionContext
+
+  var bindingFuture: Option[scala.concurrent.Future[akka.http.scaladsl.Http.ServerBinding]] = None
+
+  def setUp() = {
+    def route(n: Int): Route = AkkaHttpServerInterpreter()
```

Comment:
a more tapir-way would be to create a `List[ServerEndpoint]` with 128 elements, and interpret them all at once